packages/core/parcel/README.md

```diff
@@ -626,7 +626,7 @@ See [Resolvers](#resolvers)
 ```json
 {
   "transforms": {
```

Comment:
We should probably change this in parcelrc to match as well...

Reply:
Sounds reasonable but i am not aware of the changes and where are they applied and this is why i didn't changed to not break something.

Reply:
If it is "transforms" the values inside should be "transform"
If it is "transformers" it sounds good to be "transformer" 

Basically i will change them according to what you wrote and we can discuss more further.